.github/workflows/qa-testbench.yaml

```diff
@@ -102,8 +113,8 @@ jobs:
     needs: [qa-testbench]
     if: ${{ always() }}
     env:
-      GENERATION_TEMPLATE: "${{ github.event.inputs.generation }}"
-      BRANCH_NAME: "${{ github.event.inputs.branch }}"
+      GENERATION_TEMPLATE: "${{ inputs.generation || 'Zeebe Snapshot' }}"
+      BRANCH_NAME: "${{ inputs.branch || github.ref_name || 'main' }}"
```

Comment:
:question: Is this necessary?

Reply:
Sort of, because we want to resolve the same values as in the beginning, and we've modified them since. I guess the solution would be to use different environment variables tbh, that way we wouldn't have to specify these again. We can do that after https://github.com/camunda/zeebe/issues/11021, since I expect we'll extract a bunch of these things out.